scene-format.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package turbocow;
+
+message Scene {
+  message Object {
+    int32 id;
+    string reference;         // path to file or url
+  }
+  message Material {
+    int32 object_id;
+    string type;
+  }
+  message Light {
+    int32 id;
+    string type;
+    repeated double position; // array of three numbers
+  }
+  message Camera {
+    int32 id;
+    repeated double position;
```

Comment:
Do we support only perspective camera? If that the case we should add more parameters to it. At least `fieldOfView`. `nearPlane` and `farPlane` if we want to support all basic camera parameters